internal/venice-client-common/src/main/java/com/linkedin/venice/utils/VeniceProperties.java

```diff
@@ -389,6 +389,10 @@ public List<String> getList(String key, List<String> defaultValue) {
     }
 
     String value = get(key);
+    if (value.isEmpty()) {
+      return defaultValue;
+    }
+
```

Comment:
I'm not sure this would be safe since this is used widely. We need to be careful and audit all usages of these methods